docs/release-notes/snapcraft-8-10.rst

```diff
@@ -0,0 +1,193 @@
+.. _release-8.10:
+
+Snapcraft 8.10 release notes
+============================
+
+7 July 2025
+
+Learn about the new features, changes, and fixes introduced in Snapcraft 8.10.
+
+
+Requirements and compatibility
+------------------------------
+
+See :ref:`reference-system-requirements` for information on the minimum hardware and
+installed software.
+
+
+What's new
+----------
+
+Snapcraft 8.10 brings the following features, integrations, and improvements.
+
+
+.NET plugin (v2)
+~~~~~~~~~~~~~~~~
+
+The :ref:`craft_parts_dotnet_v2_plugin` is now available for core24 snaps. We've
+introduced a new version of the plugin that is easier to use and configure.
+
+Older bases will continue to use the :ref:`previous version <craft_parts_dotnet_plugin>`
+of the dotnet plugin.
+
+Init profile for ROS 2 applications
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+A new init profile, ``ros2``, has been added. This profile makes it easier for developers
+to create a ROS 2 snap.
+
+
+Improved ``test`` command
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The ``CRAFT_ARTIFACT`` environment variable can now be used in tests and contains the
+full path to the snap file being tested.
+
+Additionally, the user experience of the ``test`` command has been cleaned up and
+improved.
+
+
+Minor features
+--------------
+
+Snapcraft 8.10 brings the following minor changes.
+
+
+``CMAKE_PREFIX_PATH`` for the GNOME extension
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The GNOME extension now sets the ``CMAKE_PREFIX_PATH`` in the build-environment to the
+stage directory and the GNOME SDK snap.
```

Comment:
```suggestion
During the build step, the GNOME extension now points the ``CMAKE_PREFIX_PATH`` environment variable to the GNOME SDK snap inside the stage directory.
```

After reviewing the commit, I believe this is correct.

However, I'd like to urge making the _value_ more palpable to the user. Let's mention that this fixes multiple active snaps, like you did in #5461.

Reply:
Good idea, I've rewritten it to talk about the value to the user: https://github.com/canonical/snapcraft/pull/5575/commits/b74209193b1fba0ea13b8ccf46723cc5fd7a62fe

It does point to the GNOME SDK snap and stage directory separately.